seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafka/kafkasource_earliest_to_console.conf

```diff
@@ -69,6 +69,12 @@ sink {
             ]
           }
         ]
+        row_rules = [
+                {
+                  rule_type = MIN_ROW
```

Comment:
Could you add `rule_type = MAX_ROW` too to limit the actually read data count? Also please set `MIN_ROW` and `MAX_ROW` value as equals.